git/merge-a-branch-as-a-single-commit.md

```diff
@@ -0,0 +1,12 @@
+# Merge all changes from another branch as a single commit:
+
+Let's say you want to merge "my-feature" (which contain 10 commits :D ) into develop:
+
+    git checkout develop
+    git merge --squash my-feature
```

Comment:
I think `rebase -i` is even better, as it is more flexible in what you can do with the commits.